src/types/theme.d.ts

```diff
@@ -1,9 +1,13 @@
 import '@emotion/react';
+import {themeFonts} from '@/theme';
 
 /**
  * 테마 타입 정의
  * 추후에 색상을 추가할 때 이 타입을 참고하여 추가해야 합니다.
  * @author 홍규진
+ *
+ * 테마 타입에 폰트를 추가하였습니다.
+ * @author 이정선
  */
```

Comment:
```suggestion
/**
 * 테마 타입 정의
 * 추후에 색상 및 폰트를 추가할 때 이 타입을 참고하여 추가해야 합니다.
 * @author 홍규진, 이정선
 */
```

계속 리뷰가 쌓이는 구조로 작성하게 되면 이해하기 어렵기 때문에 다음과 같이 작성해주면 처음 보더라도 이해하기 쉬울 거 같아요! 😄

Reply:
네 전부 수정하겠습니다!